packages/vscode-extension/src/devices/AndroidEmulatorDevice.ts

```diff
@@ -106,10 +114,20 @@ export class AndroidEmulatorDevice extends DeviceBase {
     }
   }
 
-  private async ensureOldEmulatorProcessExited() {
+  public static async ensureOldEmulatorProcessExited(avdId: string) {
```

Comment:
Make it private please. Static is fine but I'm personally not a fan of it.

Reply:
It's static because we use it in `removeEmulator ` function which is not a member of `AndroidEmulatorDevice `.